modules/i3s/src/lib/parsers/parse-i3s-tile-content.js

```diff
@@ -27,7 +35,17 @@ export async function parseI3STileContent(arrayBuffer, tile, tileset, options) {
 
   if (tile.textureUrl) {
     const url = getUrlWithToken(tile.textureUrl, options.token);
-    tile.content.texture = await load(url, ImageLoader);
+    const loader = FORMAT_LOADER_MAP[tile.textureFormat] || ImageLoader;
+    tile.content.texture = await load(url, loader);
+    if (loader === CompressedTextureLoader) {
+      tile.content.texture = {
+        compressed: true,
+        mipmaps: false,
```

Comment:
The compressed textures do not have mipmaps?

Reply:
This flag is used in "luma" to check if we have to run `this.gl.generateMipmap(this.target);`. For compressed texture `generateMipmap` cause error.

Reply:
Ideally luma should not generate mipmaps if `compressed` is true. Maybe add a comment explaining that this is what it means.